raspibolt-pulse-switch.sh

```diff
@@ -125,6 +129,15 @@ fi
 network_rx=$(ifconfig ${network_name} | grep 'RX packets' | awk '{ print $6$7 }' | sed 's/[()]//g')
 network_tx=$(ifconfig ${network_name} | grep 'TX packets' | awk '{ print $6$7 }' | sed 's/[()]//g')
```

Comment:
```suggestion
network_rx=$(ip -h -s link show dev eth0 | grep -A1 RX | tail -1 | awk '{print $1}')
network_tx=$(ip -h -s link show dev eth0 | grep -A1 TX | tail -1 | awk '{print $1}')
```

A suggestion using `ip` instead of depracted `ifconfig`. It works but I'm sure there is a much cleaner way to get the string from the command than the piped commands I used, please amend as much as you want.

I noted that the RX and TX package sizes are a bit differnt bewteen `ifconfig` and `ip` on my node:
|   | ifconfig | ip |
|--|--------|----|
|RX|37|76|
|TX|40|82|